src/sonic-yang-mgmt/yang-models/sonic-head.yang

```diff
@@ -8,18 +8,40 @@ module sonic-head {
     contact "lnos_coders@linkedin.com";
 
     description "Head yang Module for SONiC OS";
-    
+
     revision 2019-07-01 {
```

Comment:
What is the significant of this revision?  Do we need make changes here after the changes in this PR?

Reply:
Ideally each PR in github master should have new revision number. But since it is not merged in MSFT github yet. I kept same revision number for all PR till first merge.